docs/en/tutorials/backends_support.md

```diff
@@ -0,0 +1,153 @@
+# Tutorial 7: Backends Support
+We support different file client backends: Disk, Ceph and IMDB, etc. Here is an example of Ceph loading and saving modification in config.
```

Comment:
Here is an example of how to modify configs for Ceph-based data loading and saving.